festival/src/main/java/likelion/festival/notification/service/NotificationService.java

```diff
@@ -0,0 +1,73 @@
+package likelion.festival.notification.service;
+
+import likelion.festival.notification.dto.NotificationRequestDto;
+import likelion.festival.notification.dto.NotificationResponseDto;
+import likelion.festival.notification.entity.Notification;
+import likelion.festival.notification.entity.NotificationType;
+import likelion.festival.global.exception.WrongNotificationId;
+import likelion.festival.notification.repository.NotificationRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+@Service
+public class NotificationService {
+
+  private final NotificationRepository notificationRepository;
+
+  public NotificationResponseDto readNotification(Long id) {
+    Notification notification = notificationRepository.findById(id)
+        .orElseThrow(() -> new WrongNotificationId());
+
+    NotificationResponseDto notificationResponseDto = NotificationResponseDto.toDto(notification);
+
+    return notificationResponseDto;
+  }
+
+  public List<NotificationResponseDto> readNotificationAll(NotificationType notificationType) {
+    List<NotificationResponseDto> notificationResponseDtos = new ArrayList<>();
+    if (notificationType == null) {
+      List<Notification> notifications = notificationRepository.findAll();
+      for (Notification notification : notifications) {
+        NotificationResponseDto notificationResponseDto = NotificationResponseDto.toDto(notification);
+        notificationResponseDtos.add(notificationResponseDto);
+      }
+      return notificationResponseDtos;
```

Comment:
스트림을 이용하는 것도 방법이겠네요~
```suggestion
      return notificationRepository.findAll().streams()
          .map(it -> NotificationResponseDto.toDto(it))
          .collect(Collectors.toList());
```